selfdrive/car/toyota/radar_interface.py

```diff
@@ -42,17 +42,22 @@ def __init__(self, CP):
     self.rcp = None if CP.radarUnavailable else _create_radar_can_parser(CP.carFingerprint)
     self.trigger_msg = self.RADAR_B_MSGS[-1]
     self.updated_messages = set()
+    self.f = open("/tmp/radar_data.txt", "w")
```

Comment:
This intentional?

Reply:
Oh no, that’s some leftover from debugging. Good catch!